src/Sulu/Bundle/SearchBundle/DependencyInjection/Configuration.php

```diff
@@ -24,17 +24,11 @@ public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder();
         $treeBuilder->root('sulu_search')
+            ->addDefaultsIfNotSet()
             ->children()
-                ->scalarNode('adapter_id')->defaultValue('sulu_search.adapter.zend_lucene')->end()
-                ->arrayNode('adapters')
-                    ->addDefaultsifNotSet()
-                    ->children()
-                    ->arrayNode('zend_lucene')
-                        ->addDefaultsifNotSet()
-                        ->children()
-                            ->scalarNode('basepath')->defaultValue('%kernel.root_dir%/data')->end()
-                        ->end()
-                    ->end()
+                ->scalarNode('structure_index_name')
+                    ->info('Name of the search index to use for structures (this should change depending on the sulu context)')
```

Comment:
It should not change .. if contexts were actually working then this should be a map.